modules/azure-gaming/game-dev-vmss/main.bicep

```diff
@@ -0,0 +1,247 @@
+@description('Deployment Location')
+param location string = resourceGroup().location
+
+@description('Name of VMSS Cluster')
+param vmssName string
+
+@description('GameDev Sku')
+param vmssSku string = 'Standard_D4ds_v4'
```

Comment:
can we provide a list of @allowed() values?

Reply:
One thing to keep in mind with `@allowed` is that the list needs to be fairly large - to avoid blocking users when they can't deploy (or don't have quota) for a specific size in a specific region.  And when we have to remove items from the list (or add) we'd need to update the module.

Reply:
I can not find any particular examples of vmss deployment via ARM which uses an allowed list here... I do know if you provide it the wrong item, the error will be the list of accepted values

Reply:
going to resolve this as it does not seem to be a recommend practice...